network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -0,0 +1,65 @@
+          <p class="mb-0 h4-heading">*privacy not&nbsp;included</p>
+        </div>
+        <div class="social d-flex align-items-center">
+          <a class="social-button social-button-fb" href="#TODO"><span class="sr-only">Facebook</a></a>
+          <a class="social-button social-button-twitter" href="#TODO"><span class="sr-only">Twitter</a></a>
+          <a class="social-button social-button-email" href="#TODO"><span class="sr-only">Email</a></a>
+          <a class="d-none d-sm-block btn btn-blue ml-3" href="https://donate.mozilla.org">Donate</a>
+        </div>
+      </div>
+    </header>
+    <div class="primary-nav mb-3">
+      <div class="container">
+        <a class="nav-home" href="/privacynotincluded">Home</a>
+        <a class="nav-TODO" href="#TODO">Toys & Games</a>
+        <a class="nav-TODO" href="#TODO">Smart Home</a>
+        <a class="nav-TODO" href="#TODO">Entertainment</a>
+        <a class="nav-TODO" href="#TODO">Wearables</a>
+        <a class="nav-TODO" href="#TODO">Health & Exercise</a>
+        <a class="nav-TODO" href="#TODO">Pets</a>
+        <a class="nav-about" href="/privacynotincluded/about">About the Guide</a>
+      </div>
+    </div>
+
+    <div class="pb-5">{% block guts %}{% endblock %}</div>
+
+    <footer class="mofo-footer">
+      <div class="container">
+        <ul class="list-unstyled footer-links row justify-content-center">
+          <li class="col-auto"><a class="footer-link-email" href="mailto:network@mozillafoundation.org">Email</a></li>
+          <li class="col-auto"><a class="footer-link-twitter" href="https://twitter.com/mozilla">Twitter</a></li>
+          <li class="col-auto"><a class="footer-link-cc-license" href="https://creativecommons.org/licenses/by/4.0">License</a></li>
+          <li class="col-auto"><a class="footer-link-participation-guidelines" href="https://www.mozilla.org/about/governance/policies/participation/">Participation Guidelines</a></li>
+          <li class="col-auto"><a class="footer-link-legal" href="https://mozilla.org/en-US/about/legal/">Legal</a></li>
+          <li class="col-auto"><a class="footer-link-privacy" href="https://mozilla.org/en-US/privacy/websites/">Privacy</a></li>
+          <li class="col-auto"><a class="footer-link-donate" id="donate-footer-btn" href="https://donate.mozilla.org?utm_source=foundation.mozilla.org&amp;utm_medium=referral&amp;utm_content=footer" target="_blank" rel="noopener noreferrer">Donate</a></li>
+        </ul>
+        <div class="org-info mozilla">
+          <div class="logo-container"><a class="logo" href="https://mozilla.org"></a></div>
+          <p>Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>.</p>
+        </div>
+      </div>
+    </footer>
+  </body>
+</html>
+<script src="/_js/bg-main.compiled.js"></script>
```

Comment:
technically this is invalid HTML since nothing's allowed to be between `</body></html>` or after `</html>` - given the current state of browsers, could we just move it to before `</body>` and mark it as `defer` so that it doesn't actually load until after the DOM has been fully parsed?

Reply:
Shouldn't matter, but sure.